main/res/values/strings.xml

```diff
@@ -747,23 +747,23 @@
     <string name="init_debug_note">c:geo can generate a lot of debugging information. Activating this setting will affect the stability of c:geo and is only recommended if you are asked to send a log to the developers. Please keep debug logging disabled during normal usage of c:geo!</string>
     <string name="init_debug">Activate debug log</string>
     <string name="init_dbonsdcard_title">Database location</string>
-    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or in a directory on your user storage. Having it on the user storage may free some memory, but you may lose a bit of performance and c:geo will not work if the storage is not accessible or you remove the database file.</string>
+    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or into a folder on your user storage. We recommend to only activate this option if you need user access to your database file.</string>
```

Comment:
IMHO the rationale for using user storage for the database is more something like "regular access" or "frequent access" to the database. If you only need it once in a while you are better of by doing a backup.

Reply:
rephrased to "regular manual access"